packages/sdks/src/components/block/block.helpers.ts

```diff
@@ -77,7 +77,7 @@ export const getRepeatItemData = ({
         $index: index,
         $item: item,
         [itemNameToUse]: item,
-        [`$${itemNameToUse}Index`]: index,
```

Comment:
this looks like a breaking change, no?
should we not support both so we don't break existing content?